native/database/db_test.go

```diff
@@ -0,0 +1,84 @@
+package database
+
+import (
+	"testing"
+)
+
+func TestInsertTransaction(t *testing.T) {
+	err := InitDB(":memory:") // in-memory database
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tx := Transaction{
+		BtcTxID: 1,
+		RawTx:   "raw-transaction-hex",
+		Status:  StatusPending,
+	}
+
+	err = InsertTransaction(tx)
+	if err != nil {
+		t.Errorf("InsertTransaction() error = %v", err)
+	}
+}
+
+func TestGetPendingTransactions(t *testing.T) {
+	err := InitDB(":memory:") // in-memory database
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	transactions := []Transaction{
+		{BtcTxID: 1, RawTx: "tx1-hex", Status: StatusPending},
+		{BtcTxID: 2, RawTx: "tx2-hex", Status: StatusBroadcasted},
+		{BtcTxID: 3, RawTx: "tx3-hex", Status: StatusPending},
+	}
+	for _, tx := range transactions {
+		err = InsertTransaction(tx)
+		if err != nil {
+			t.Fatal(err)
+		}
+	}
+
+	pendingTxs, err := GetPendingTransactions()
+	if err != nil {
+		t.Errorf("GetPendingTransactions() error = %v", err)
+	}
+
+	if len(pendingTxs) != 2 {
+		t.Errorf("Expected 2 pending transactions, got %d", len(pendingTxs))
+	}
```

Comment:
```suggestion
	import "gotest.tools/assert"
	...
	
	assert.Equal(t, 2, len(pendingTxs), "expected 2 pending transactions")
	}
```

Reply:
Note, we don't need to add "got ..." in the message - the `assert.Equal` will print the diff.

Reply:
Done